packages/oas-to-har/src/lib/style-formatting/index.js

```diff
@@ -0,0 +1,117 @@
+const stylize = require('./style-serializer');
+
+// Certain styles don't support empty values, This function tracks that list
+function shouldNotStyleEmptyValues(parameter) {
+  return ['simple', 'spaceDelimited', 'pipeDelimited', 'deepObject'].includes(parameter.style);
+}
+
+// Note: This isn't necessarily part of the spec. Behavior for the value 'undefined' is, well, undefined.
+//   This code makes our system look better. If we wanted to be more accurate, we might want to remove this,
+//   restore the un-fixed behavior for undefined and have our UI pass in empty string instead of undefined.
+function removeUndefinedForPath(value) {
```

Comment:
Can you use https://github.com/readmeio/api-explorer/blob/next/packages/oas-to-har/src/lib/remove-undefined-objects.js?

Reply:
Spent some time testing this, and I can't because the purpose is not to remove undefined, but to make undefined empty strings.